packages/polar/src/internal/core/runtime-env.ts

```diff
@@ -51,7 +51,13 @@ export class Environment implements PolarRuntimeEnvironment {
   ) {
     log("Creating RuntimeEnv");
 
-    const ncfg = config.networks[runtimeArgs.network];
+    let ncfg = config.networks[runtimeArgs.network];
+    if (runtimeArgs.network === "local") {
+      ncfg = {
+        endpoint: 'http://localhost:1337/',
```

Comment:
make this configurable instead of hardcode.

Reply:
not possible at this moment

Reply:
move it constants file then.